src/org/zaproxy/zap/extension/tokengen/ZapAddOn.xml

```diff
@@ -9,6 +9,9 @@
 	<![CDATA[
 	Minor code changes.<br>
 	Added help file.<br>
+	Issue 2338: Allow dynamic timeout adjustment to combat read timeout issues.<br>
+	Ensure initial dialog is properly sized.
```

Comment:
Should have a `br` at the end of the line(?). (Same comment for the following line.)